app/src/main/java/io/gnosis/safe/ui/splash/SplashActivity.kt

```diff
@@ -35,11 +38,19 @@ class SplashActivity : BaseActivity() {
                         }
                     }.show(supportFragmentManager, TermsBottomSheetDialog::class.simpleName)
                 }
+                is SplashViewModel.ShowButton -> {
+                    binding.continueButton.visibility = View.VISIBLE
```

Comment:
you could use visible extension from svalinn: binding.continueButton.visible(true)